signals.go

```diff
@@ -70,7 +70,7 @@ func (h *signalHandler) forward(process *libcontainer.Process, tty *tty, detach
 			h.notifySocket.run(pid1)
 			return 0, nil
 		} else {
-			go h.notifySocket.run(0)
+			h.notifySocket.run(os.Getpid())
```

Comment:
question: does it matter here that we set PID to `runc` process ? Cant we always set it to the container's process instead ?

Reply:
if we keep runc running, the container's process won't be a direct child of systemd and systemd won't get the SIGCHLD once it terminates.  Differently, it will get the SIGCHLD for the runc process and can link it to the service being terminated.